src/antworld/route_ardin.cc

```diff
@@ -275,7 +275,7 @@ void RouteArdin::render(const Pose2<meter_t, degree_t> &pose, meter_t height) co
 
     glBindVertexArray(m_OverlayVAO);
 
-    glTranslatef(static_cast<GLfloat>(pose.x().value()), static_cast<GLfloat>(pose.y().value()), 0.0f);
+    glTranslatef(static_cast<GLfloat>(pose.x().value()), static_cast<GLfloat>(pose.y().value()), 0.01f);
```

Comment:
I'm guessing you are hardcoding the height to 10cm here, right? Shouldn't z be set to ``height`` instead?

Reply:
So, (deprecated but convenient) OpenGL provides you with matrix stack onto which you can pop and push transforms. At the top of these functions we push the height z-transform before doing any (essentially 2D) drawing. This additional translate is just to make sure the arrow renders above the route

Reply:
Ahhh I see. Makes sense. There's so much I don't know about OpenGL :laughing: